apps/google-analytics-4/src/locations/ConfigScreen.spec.tsx

```diff
@@ -9,14 +9,24 @@ jest.mock('@contentful/react-apps-toolkit', () => ({
 }));
 
 describe('Config Screen component', () => {
-  it('Component text exists', async () => {
-    const { getByText } = render(<ConfigScreen />);
+  let renderedComponent: RenderResult;
 
-    // simulate the user clicking the install button
+  beforeEach(() => {
+    renderedComponent = render(<ConfigScreen />);
+  });
+
+  it('shows config screen', async () => {
+    const { getByText } = renderedComponent;
+
+    expect(getByText('About Google Analytics for Contentful')).toBeInTheDocument();
+  });
+
+  it('can be installed', async () => {
+    const { getByText } = renderedComponent;
+
+    // install the app
     await mockSdk.app.onConfigure.mock.calls[0][0]();
 
-    expect(
-      getByText('Welcome to your contentful app. This is your config page.')
-    ).toBeInTheDocument();
+    expect(getByText('About Google Analytics for Contentful')).toBeInTheDocument();
```

Comment:
This test would pass if we do not trigger the install. Should there be any feedback about the installation state within the app>

Reply:
Yes 👍 . I will address this in a subsequent PR. For now these tests are still mainly just boilerplate.